app/models/plugins/call_tool.rb

```diff
@@ -97,4 +99,10 @@ def targets_are_valid
       end
     end
   end
+
+  def target_countries_are_present
+    targets.select { |t| t.country_code.blank? }.each_with_index do |_, index|
+      errors.add(:targets, "Country can't be blank (row #{index + 1})")
```

Comment:
I'm not sure I understand your question. I'm including the row in the error message to help the campaigner uploading the CSV know which row is the culprit.